src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -19,17 +19,69 @@ const StatCard = ({ number, secondaryNumber, statement }: StatCardProps) => (
   </div>
 );
 
+type AxisProp = {
+  x: string | number;
+  y: number;
+};
+
+type BarStatProps = {
+  axes: AxisProp[];
+  legend: { title: string; subtitle: string };
+  position: string;
```

Comment:
I think "orientation" might be a better name for this. "Position" makes me think of "where this should be placed", such as "top", "bottom", "bottom-left". I think "orientation" is a more common word for whether something is horizontal or vertical.

Could you also set the type to `"horizontal" | "vertical"` instead of `string`? This would give is much tighter type checking, since all other string values would be invalid.